rfcs/text/0000-roadmaps+planning.md

```diff
@@ -0,0 +1,432 @@
+- [Example use in the Scaled Agile Framework](https://www.scaledagileframework.com/spikes/)
+
+**Milestones** represent a four week timeline/duration in a Release.
+
+**Releases** are composed of three milestones and occur on a quarterly cadence.
+
+## Detailed Design for Roadmaps
+
+The Product Roadmap creates a shared understanding across the DCP community and Users of the DCP for investments and direction for the next year. The Roadmap process MUST occur on an annual basis. 
+
+Themes and Objectives from the Roadmap are the primary source of priorities for planning Releases each quarter.
+
+### Timebox for Roadmaps
+
+All timeboxes are noted as business days. Business days are included regardless of national holidays.
+
+Overlapping timeboxes denote concurrent work.
+
+Timeboxes with a range of days indicate that the work MAY be completed at any point during the timebox, but MUST be completed no later than the final day of the timebox.
+
+**Note**: *Input from the Technical Architecture team is integrated on a quarterly basis during Release planning.*
+
+Day 1 in the Roadmap timeline begins in the third quarter.
+
+* **Day 1 - Day 30: [Gathering and reviewing input](#gathering-and-reviewing-input)**
+  * Prerequisite: Science Roadmap
+  * Prerequisite: Technical Objectives RFC
+  * Day 1-30: DCP User Survey 
+  * Day 1-10: Roadmap Retrospective
+* **Day 1 - Day 45: [Drafting the roadmap](#drafting-the-roadmap)**
+  * Day 1-10: Identify a Roadmap Shepherd
+  * Day 30-35: Create themes
+  * Day 36: Collate themes
+  * Day 37-45: Refine themes
+* **Day 45 - Day 60: Receiving feedback**
+  * Day 45: Publish draft Roadmap
+  * Day 46-55: Community Review period 
+  * Day 56-60: Complete final edits
+* **Day 60 - Day 70: [Iterating with Oversight](#iterating-with-oversight)**
+  * Day 60: Send Roadmap to Oversight
+  * Day 61-70: Review and edit roadmap with Oversight
+  * Day 70: Receive Roadmap approval from Oversight
+  * Day 70: Communicate approved Roadmap 
+
+
+### Gathering and Reviewing Input
+[Gathering and Reviewing Input]:#gathering-and-reviewing-input
+
+#### Science Roadmap
+
+Oversight MUST provide an annual Science Roadmap to Project Leads to be reviewed at the next Oversight meeting following its receipt.
+
+Project Leads MUST clarify outstanding questions about the Science Roadmap with Oversight within two weeks.
+
+#### Technical Objectives RFC
+
+Project Leads MUST review the [Technical Objectives RFC](#technical-architecture-objectives) from the most recent quarter to understand outstanding technical concerns.
+
+#### DCP User Survey and UX Insights
+
+User Experience (UX) MUST create a DCP user survey on an annual basis to collect input for the Product Roadmap. The audience for the survey are the Users of the DCP.
+
+UX MUST review the proposed survey with the DCP community and incorporate suggestions as appropriate.
```

Comment:
Hi, and apologies for only finding time to review this now.
I do agree that we need to also incorporate and act on user feedback and real life evidence as it comes from other sources like ongoing user evaluation and research. A annual survey can give an overall idea of the current general attitudes, can provide some additional quantitative substance to our other user experience findings, but it's not the way to inform roadmapping and provide timely feedback which can be incorporated into the development process.

There need to be cycles of delivery and evaluation/research, and their cadence - probably with every major release of a functionality, which addresses a clear user goal. 

So my question is: what happens if the functionality which is in the **annual** roadmap is not answering user needs? What happens with ongoing feedback which comes from usability testing?

I have examples, which I would be interested to see used for this suggested roadmapping process. Right now we are working on two reports: one is a piece of research on Researcher with a pipette, and their workflows, needs and expectations, when it comes to the HCA DCP (it's an impressive piece of work, which can guide future priorities for this persona).

The other one is the ongoing feedback we have had from data contributors, which we will also triangulate with a regular survey with data contributors to get feedback on data submission. We additionally know the problems the data wranglers as users have when submitting data. 

Specifically with the second piece of UX feedback, we already know SO MANY of the problems and I am now in the process of specifying this wrangler project backoffice, which would allow wranglers to do their work efficiently. This is going to have impact on end users as well in the future, and we are going to also evaluate this as it gets developed, and act on feedback.

This hasn't been on the roadmap so far (well as far as a I know), .. so how will it look on the overall roadmap?

I guess my general comments are - the current outline of the roadmapping process doesn't show how it can be flexible enough to support real life evidence, incorporate user research and user feedback. (the way it looks now, at least). It doesn't seem to incorporate the iterative principle of user experience design.

![230e3eb882eb8379887eeef3b553a4f0](https://user-images.githubusercontent.com/19713125/62790449-111ccd00-bac3-11e9-909a-5d9e28852aae.jpg)